src/MSBuild/MSBuild.csproj

```diff
@@ -36,7 +36,7 @@
     <NoPackageAnalysis>true</NoPackageAnalysis>
     <AddAppConfigToBuildOutputs>false</AddAppConfigToBuildOutputs>
 
-    <DebugType Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">full</DebugType><!-- Work around arcade stomping on symbols for same-program-different-arches. -->
+    <DebugType Condition="'$(Platform)' == 'x64'">full</DebugType><!-- Work around arcade stomping on symbols for same-program-different-arches. -->
```

Comment:
Can you fix my incorrect comment? point to AfterSigning.targets too, and describe how it works for x64 and non-x64 flavors.

Reply:
Here's my working comment. I'm not sure what to put in place of 'more compatible', can you provide some input for that? or reword as necessary.
 ```Setting DebugType here goes hand in hand with AfterSigning.targets. This is to prompt the x64 build to produce a 'full' .pdb that's `more compatible` then 'portable' and 'embedded' .pdbs. This doesn't get set on 32 bit architecture, which will default to 'embedded' and 'pdb2pdb' will convert those as needed. See github pull #5070 for context. ```

Reply:
looks ok to me, but use a full URL for the PR. and a full local path for aftersigning.